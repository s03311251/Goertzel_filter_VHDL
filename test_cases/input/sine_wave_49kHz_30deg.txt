2FFF
37A4
3D10
3FC1
3F74
3C33
364A
2E49
24F0
1B20
11C5
09C1
03D4
008E
003C
02E7
084F
0FF1
1915
22E0
2C66
34C1
3B28
3F02
3FF1
3DDF
38FE
31C3
28DC
1F20
157A
0CD0
05F4
018C
0000
0177
05CD
0C99
1539
1EDC
289A
318A
38D3
3DC6
3FED
3F13
3B4C
34F5
2CA5
2325
1958
102C
087D
0304
0045
0081
03B4
0990
1188
1ADC
24AC
2E0C
3619
3C12
3F67
3FC9
3D2D
37D2
303B
271C
1D52
13C9
0B65
04F2
010A
000B
020E
06E1
0E11
16F1
20AB
2A55
3306
39ED
3E63
3FFF
3E97
3A50
338F
2AF7
2157
1796
0EA0
074D
024C
0016
00E0
0498
0AE3
132B
