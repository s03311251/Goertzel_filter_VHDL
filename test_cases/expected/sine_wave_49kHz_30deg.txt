FFEC5A80
FFEC0140
